### PR TITLE
[FIX] empty spel template should evaluate to empty string

### PR DESCRIPTION
**Pull request by mproch**

### CODE REVIEW

File: engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/SpelExpression.scala [hunk 1]
```diff
@@ -161,6 +161,9 @@ class SpelExpressionParser(parser: org.springframework.expression.spel.standard.
     }
   }
 
+  private def shouldUseNullExpression(original: String): Boolean
+  = flavour != Template && StringUtils.isBlank(original)
```
Comment (alien11689) [inline engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/SpelExpression.scala]: strange formatting ;)

File: engine/interpreter/src/test/scala/pl/touk/nussknacker/engine/spel/SpelExpressionSpec.scala [hunk 2]
```diff
@@ -510,6 +510,7 @@ class SpelExpressionSpec extends FunSuite with Matchers with EitherValues {
   test("evaluates expression with template context") {
     parseOrFail[String]("alamakota #{444}", ctx, SpelExpressionParser.Template).evaluateSyncToValue[String]() shouldBe "alamakota 444"
     parseOrFail[String]("alamakota #{444 + #obj.value} #{#mapValue.foo}", ctx, SpelExpressionParser.Template).evaluateSyncToValue[String]() shouldBe "alamakota 446 bar"
+    parseOrFail[String]("", ctx, SpelExpressionParser.Template).evaluateSyncToValue[String]() shouldBe ""
```
Comment (alien11689) [inline engine/interpreter/src/test/scala/pl/touk/nussknacker/engine/spel/SpelExpressionSpec.scala]: Why don't you create separate test with name for such case?

File: engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/SpelExpression.scala (hunk 1 above)
Comment (mproch) [reply engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/SpelExpression.scala]: done

File: engine/interpreter/src/test/scala/pl/touk/nussknacker/engine/spel/SpelExpressionSpec.scala (hunk 2 above)
Comment (mproch) [reply engine/interpreter/src/test/scala/pl/touk/nussknacker/engine/spel/SpelExpressionSpec.scala]: done

**Pull request merged**